### PR TITLE
[Security]: Add HTML sanitizer for WYSIWYG fields

### PR DESCRIPTION
**Pull request by lukmzig**

### CODE REVIEW

File: models/DataObject/ClassDefinition/Data/Wysiwyg.php [hunk 1]
```diff
@@ -48,6 +51,15 @@ class Wysiwyg extends Data implements ResourcePersistenceAwareInterface, QueryRe
      */
     public string|int $maxCharacters = 0;
 
+    public static function getWysiwygSanitizer(): HtmlSanitizer
+    {
+        if (!isset(self::$pimcoreWysiwygSanitizer)) {
+            self::$pimcoreWysiwygSanitizer = \Pimcore::getContainer()->get(Text::PIMCORE_WYSIWYG_SANITIZER_ID);
+        }
+
+        return self::$pimcoreWysiwygSanitizer;
```
Comment (jdreesen) [inline models/DataObject/ClassDefinition/Data/Wysiwyg.php]:
```suggestion
        return self::$pimcoreWysiwygSanitizer ??= \Pimcore::getContainer()->get(Text::PIMCORE_WYSIWYG_SANITIZER_ID);
```

File: models/Document/Editable/Wysiwyg.php [hunk 2]
```diff
@@ -19,19 +19,31 @@
...
      */
     protected ?string $text = null;
 
+    public static function getWysiwygSanitizer(): HtmlSanitizer
+    {
+        if (!isset(self::$pimcoreWysiwygSanitizer)) {
+            self::$pimcoreWysiwygSanitizer = \Pimcore::getContainer()->get(Text::PIMCORE_WYSIWYG_SANITIZER_ID);
+        }
+
+        return self::$pimcoreWysiwygSanitizer;
```
Comment (jdreesen) [inline models/Document/Editable/Wysiwyg.php]:
```suggestion
        return self::$pimcoreWysiwygSanitizer ??= \Pimcore::getContainer()->get(Text::PIMCORE_WYSIWYG_SANITIZER_ID);
```

File: models/DataObject/ClassDefinition/Data/Wysiwyg.php [hunk 3]
```diff
@@ -105,6 +117,10 @@ public function getDataForResource(mixed $data, DataObject\Concrete $object = nu
      */
     public function getDataFromResource(mixed $data, DataObject\Concrete $object = null, array $params = []): ?string
     {
+        $helper = self::getWysiwygSanitizer();
+        if (is_string($data)) {
+            $data = $helper->sanitize(html_entity_decode($data));
+        }
```
Comment (jdreesen) [inline models/DataObject/ClassDefinition/Data/Wysiwyg.php]:
```suggestion
        if (is_string($data)) {
            $data = self::getWysiwygSanitizer()->sanitize(html_entity_decode($data));
        }
```

Or at least only get it when necessary:

```suggestion
        if (is_string($data)) {
            $helper = self::getWysiwygSanitizer();
            $data = $helper->sanitize(html_entity_decode($data));
        }
```

Comment (lukmzig) [reply models/DataObject/ClassDefinition/Data/Wysiwyg.php]: @jdreesen thank you for these suggestions! Should be adapted now.

**Pull request merged**